.github/workflows/release.yml

```diff
@@ -53,11 +53,12 @@ jobs:
     if: github.event_name == 'release'
     steps:
       - uses: actions/checkout@v2
-      - name: publish to crates.io
-        uses: actions-rs/cargo@v1
```

Comment:
You can't use `working-directory` with a `uses`, so I just went back to using the built in `cargo`